Creating Test Playbooks Checking Empty Inputs (#26048)
demisto/content: Packs/Cybereason/ReleaseNotes/2_1_6.md

#### Playbooks

##### Unisolate Endpoint - Cybereason

> - Fixed a minor bug in the Unisolate Endpoint - Cybereason where the input.hostname is defined returns true when the actual hostname field is empty.

Reviewer: ```suggestion
Fixed a minor bug in the Unisolate Endpoint - Cybereason where the input.hostname returned true when the actual hostname field was empty.
```
